API.md

```diff
@@ -122,11 +109,14 @@ So, to recap, your plugin might look a little like this:
 
 ```js
 const plugin = opts => {
-  return files => files.map(file => file.contents = opts)
+  return files => files.map(file => {
+    file.contents = opts | 'test'
```

Comment:
@pup Shouldn’t this be `||` instead of `|`

Reply:
whoops, it should!